pkg/apis/bundle/v1alpha1/builder_types.go

```diff
@@ -91,4 +91,13 @@ type ComponentBuilder struct {
 	// inline process, the data is inserted into a generated config map before
 	// being added to the objects. A ConfigMap is generated per-filegroup.
 	RawTextFiles []FileGroup `json:"rawTextFiles,omitempty"`
+
+	// BuildTags are tags to apply to the component during component-build time.
+	// The primary purpose of BuildTags is so that built-components can be
+	// exported to specific directories. It is stored as a comma-separated value
+	// in an annotation on the built-component.
+	//
+	// Build-tags should follow the same rules as Kubernetes names. See
+	// https://kubernetes.io/docs/concepts/overview/working-with-objects/names/
+	BuildTags []string `json:"buildTags,omitempty"`
```

Comment:
Why not just use annotations?

Reply:
This gets stuffed into a built-in annotation. I like the explicitness here in the Builder.